test/unit/store/state_db_test.exs

```diff
@@ -0,0 +1,135 @@
+defmodule Unit.Store.StateDb do
```

Comment:
We should add a test for `get_latest_state` (at least with 0, 1, and many states).

Reply:
Done!